stages/org.osbuild.testmac

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+"""
+Enable osbuild mac test service
+
+Creates a test service with the osbuild_mac_t label and tests this services permissions.
+"""
+
+import subprocess
+import sys
+import os
+import re
+
+def main():
+
+    pid = '/proc/' + str(os.getpid()) + '/status'
+    temp = subprocess.Popen(['grep', 'CapBnd', pid], stdout = subprocess.PIPE)
```

Comment:
Are we sure there will only be one match for this? Could other fields contain the same value? Might it make sense to iterate the lines of this file, parse out the fields, find the first line where the first field is `CapBnd` and use that? Just to be on the safe side.

Reply:
Good plan to be safe.

The docs only mention the 5 `Cap` fields, but it could change in the future :smile:

Reply:
implemented :smile: